src/app/Main/Body/QueryPage/Results/TableResults/QueryPageContent-TableResultsLayout.component.tsx

```diff
@@ -30,14 +30,8 @@ export interface TableResultsProps {
     /**
      * attributes of the columns
      */
-    columnAttributes: string[];
-    tableColumn: string;
-    tableRow: string;
-    tableOption: DisplayOptionType;
-    tables: {
-        statisticDataAbsolute: Record<string, string>[];
-        statisticDataPercent: Record<string, string>[];
-    };
+    columnNameValues: string[];
+    tableContext: TableInterface

```

Comment:
Consider not using context as a name for a prop (because it is a reserved react keyword), just table or tableData would be fine.

Reply:
Done. Commit "Little Changes"